gallery/gallery-library/src/main/java/org/silverpeas/components/gallery/Watermark.java

```diff
@@ -0,0 +1,67 @@
+package org.silverpeas.components.gallery;
+
+import org.silverpeas.core.util.StringUtil;
+
+public class Watermark {
+
+  private boolean enabled = false;
+
+  private String IPTCPropertyForHD;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "IPTCPropertyForHD" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS00116)